### DISCUSSION TITLE
Large playlist folders - playlist or request.dynamic

### DISCUSSION THREAD

**Discussion by Russsgithub** [savonet/liquidsoap#2583] [Q&A]
I've been asked to write a script that potentially has 5000 - 10000 files in a folder to play randomly.

I'm wondering if this amount of files played through ```playist()``` is going to cause problems, in which case I should probably use a request queue. I'd much prefer the simplicity of ```playlist()``` realistically, but thought it would be good to get some opinions.

Thanks

**Comment by smimram** [comment]
I did some work a while ago (#2162) and `playlist` is now currently tested with 100000 files, so that you should be fine here.

**Comment by Russsgithub** [comment]
Thanks @smimram

On 29 Aug 2022 09:20, Samuel Mimram ***@***.***> wrote:

I did some work a while ago (#2162<https://github.com/savonet/liquidsoap/issues/2162>) and playlist is now currently tested with 100000 files, so that you should be fine here.
